packages/kerosene-ui/src/hooks/useRect.ts

```diff
@@ -25,9 +25,11 @@ const DEFAULT_RECT: Rect = {
 /**
  * Custom React Hook for reading bounding rect of a DOM Element
  * @param disable When set to `true`, updating the rect will be disabled
+ * @param event Listens for specified event type and triggers update, default is "scroll"
  */
 export default function useRect(
   disable = false,
+  event = "scroll",
```

Comment:
What if we want to specify more than one event name? I think we might want this to be an array.